### ISSUE TITLE
Elevated Risk level set to Processing.

### ISSUE THREAD

**Issue by ryanr14** [woocommerce/woocommerce-gateway-stripe#818]
#### Affected ticket(s)

1889635-zen

#### What I expected

I'd found this earlier issue, https://github.com/woocommerce/woocommerce-gateway-stripe/pull/535, and like how that looked like it works. I'm not having the same experience though at the moment, and neither is a customer I had in chat.

What I expect is that if Stripe flags the transaction with an elevated risk level, meaning the Stripe account owner needs to approve the transaction, the order in WooCommerce should be On Hold and reflect the actions the above PR once showed.

#### What happened instead

Using the test card for an elevated risk for Radar, the order just goes to Processing without warning the store owner it needs reviewed in Stripe before they process the order and ship out the product.
#### Steps to reproduce the issue

Use the elevated card number here with any valid expiration date and made up CVC.

4100000000000019

You'll then see the order is Processing and appears fine in WooCommerce while in Stripe, it'll tell you to review the transaction.

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

**Comment by AashikP** [comment]
Tried checking out with the above card and the payment was blocked on Stripe, and the order was set to Failed on WooCommerce. Got a card declined note while trying to process payment on the checkout  page. Is there a step I'm missing here? @ryanr14 

![https://cld.wthms.co/RZBn9V+](https://cld.wthms.co/RZBn9V+)
*Direct Link: https://cld.wthms.co/RZBn9V*

![https://cld.wthms.co/cpgBCz+](https://cld.wthms.co/cpgBCz+)
*Direct Link: https://cld.wthms.co/cpgBCz*

I'm on WooCommerce 3.5.7 and Stripe 4.1.15

**Comment by ryanr14** [comment]
Thanks for testing @AashikP, I just double checked and left the wrong card number.

It should be 4000000000009235 to get an [elevated](https://stripe.com/docs/radar/testing) risk evaluation.

**Comment by AashikP** [comment]
Thank you for the quick response @ryanr14 . I can replicate this behavior now

![https://cld.wthms.co/PgZk4C+](https://cld.wthms.co/PgZk4C+)
*Direct Link: https://cld.wthms.co/PgZk4C*

![https://cld.wthms.co/B9sMdF+](https://cld.wthms.co/B9sMdF+)
*Direct Link: https://cld.wthms.co/B9sMdF*

**Comment by melindahelt** [comment]
2136163-zen another report  of order going to processing instead of on hold. Adding the missing bug and priority tags and will ping in slack due to the time this has been waiting here without tags.
